bofire/utils/multiobjective.py

```diff
@@ -24,12 +29,16 @@ def get_ref_point_mask(
     in the mask is 1, in case we want to minimize it is -1.
 
     Args:
-        domain (Domain): Domain for which the mask should be generated.
-        output_feature_keys (Optional[list], optional): Name of output feature keys
-            that should be considered in the mask. Defaults to None.
+        domain: Domain for which the mask should be generated.
+        output_feature_keys: Name of output feature keys
+            that should be considered in the mask. If `None` is provided, all keys
+            belonging to output features with one of the following objectives will
+            be considered: `MaximizeObjective`, `MinimizeObjective`,
+            `CloseToTargetObjective`. Defaults to None.
 
     Returns:
-        np.ndarray: _description_
+        Array ones with 1.0 for maximization and -1.0 for
```

Comment:
Reformulate to _Array of ones for maximization and array of negative ones for minimization._